cmd/sentry/main.go

```diff
@@ -48,10 +48,6 @@ func main() {
 
 	metricsExporter := metrics.NewExporterWithOptions(log, metrics.DefaultMetricNamespace, opts.Metrics)
 
-	if len(opts.TokenAudience) > 0 {
-		log.Warn("--token-audience is deprecated and will be removed in Dapr v1.14")
```

Comment:
Why is `token-audience` being removed now if it is planned to be removed in 1.14?

Reply:
It's completely unused in this service

Reply:
What I mean is, setting this option today just prints a warning and is a no-op. Should never have said that will be removed, and should have just been removed. This is a control plane service so removing CLI flags is not an issue